### PR TITLE
Rename Project (RBIBeacon)

### PR DESCRIPTION
**Pull request by cchestnut91**
Now that we own the project rebrand it with the RBI prefix throughout the Pod & project.

### CODE REVIEW

**Comment by atticoos** [comment]
We should avoid having to do maintenance to the legacy mobile targets to use this new naming. 
Can we tag and version the existing code base before we refactor it?

**Comment by cchestnut91** [comment]
This new version is tagged 0.2.0, old one was 0.1.0. Should we need to do anything else? Not entirely familiar with the details of updating Pods. I was under the assumption that the older projects could just specify `'~> 0.1.0'`

**Comment by cchestnut91** [comment]
Specifying the version will not work with using an external source (GitHub) to load the Pod. What we could do is create a `legacy/0.1.0` branch and point the older projects to that branch, and then update the master branch to the new name. Thoughts @ajwhite @brianmuse ?

**Comment by brianmuse** [comment]
Bumping this. Looks like a conflict atm. generally this is good from me tho.

**Comment by atticoos** [comment]
I believe we've determined that using tags is compatible with Podfiles pointing to repositories.

At this point I think that thread is resolved and we're all set to continue forward on this

File: README.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ To run the example project, clone the repo, and run `pod install` from the Examp
...
 ## Installation
 
-CYFBeacon is available through [CocoaPods](http://cocoapods.org). To install
+RBIBeacon is available through [CocoaPods](http://cocoapods.org). To install
 it, simply add the following line to your Podfile:
 
-    pod "CYFBeacon"
+    pod "CYFBeacon", :git => 'https://github.com/robinpowered/CYFBeacon.git'
 
 ## Author
```
Comment (brianmuse) [inline README.md]: Let's just change this to Contributors...definitely leave Victor and you can add yourself since youre maintaining now.

**Review by atticoos — APPROVED** [review]

**Comment by atticoos** [comment]
Merging given that the latest feedback was minor, and comment prior to that was looking good.

**Pull request merged**